pkg/syncer/sync.go

```diff
@@ -8,91 +8,15 @@ import (
 
 	"github.com/juju/errors"
 	"github.com/mkmik/multierror"
+	helm "helm.sh/helm/v3/pkg/action"
```

Comment:
Is this a leftover?

Reply:
no! Take into account that now we are doing:

```
		pkgCli := helm.NewPackage()
```

and I thought that `action.NewPackage` was a bit open for this context.